core/src/test/java/org/infinispan/statetransfer/ClusterTopologyManagerTest.java

```diff
@@ -462,15 +462,4 @@ public void testLeaveDuringGetTransactions() throws InterruptedException, Timeou
       log.debugf("Recovery took %s", Util.prettyPrintTime(endTime - startTime));
       assert endTime - startTime < 30000 : "Recovery took too long: " + Util.prettyPrintTime(endTime - startTime);
    }
-
-   public void testJoinerBecomesOnlyMember() {
-      // Keep only 2 nodes for this test
-      killMember(2, CACHE_NAME);
-      defineConfigurationOnAllManagers(OTHER_CACHE_NAME, new ConfigurationBuilder().read(manager(0).getDefaultCacheConfiguration()));
-
-      d2.setDiscardAll(true);
-      fork((Callable<Object>) () -> cache(1, OTHER_CACHE_NAME));
```

Comment:
Don't forget to drop this import:
[ERROR] /home/paul/infinispan/master/core/src/test/java/org/infinispan/statetransfer/ClusterTopologyManagerTest.java:13:8: Unused import - java.util.concurrent.Callable. [UnusedImports]